.github/workflows/lint.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           # External doxygen style files and MOOSE Doxygen header do not meet Platypus
           # linting requirements; exclude them from lint checks:
```

Comment:
You should update this comment to reflect codecov.sh.

Reply:
Updated!